unix_server/authorized_identities.go

```diff
@@ -69,7 +69,9 @@ func (i *PubKeyIdentity) Verify(genericCandidate interface{}, base64Conversation
 			if clientId, ok := claims["client_id"]; !ok || clientId != fmt.Sprintf("ssh3-%s", i.username) {
 				return false
 			}
-			if jti, ok := claims["jti"]; !ok || jti != base64ConversationID {
+
+			// if a JTI is specifically set, use it. If absent, then rely on expiration for replay protection
```

Comment:
Could there be a way for a proxy to remove the JTI without it being noticed and extend the expiration ?

Reply:
JWTs are signed with the client's private key so JWTs and their claims (including `jti`) can't be tampered with.